src/crimson/os/seastore/btree/fixed_kv_btree.h

```diff
@@ -1491,9 +1491,9 @@ class FixedKVBtree {
     // checking the lba child must be atomic with creating
     // and linking the absent child
     if (v.has_child()) {
-      return v.get_child_fut().safe_then(
-        [on_found=std::move(on_found), node_iter, c,
-        parent_entry](auto child) mutable {
+      return trans_intr::make_interruptible(std::move(v.get_child_fut())
+      ).si_then([on_found=std::move(on_found), node_iter, c,
+                parent_entry](auto child) mutable {
```

Comment:
Wondering whether `mutable` is needed for `on_found`, it doesn't look modified, same to the below places.